### ISSUE TITLE
Plan to support gnome 40?

### ISSUE THREAD

**Issue by Chrysostomus** [material-shell/material-shell#529]
Gnome 40 seems to bring massive changes to gnome-shell, including horizontal workspaces. Does Material-shell project have enough resources to support gnome 40? Is there some breakage or refactoring to be expected with this update?

**Comment by PapyElGringo** [comment]
It's should be okay but it's would probably be the time to remove the activities from Material Shell and replace it by our own implementation.

**Comment by Chrysostomus** [comment]
That makes sense, as the maintenance would become really difficult with the new activities.

**Comment by paranoidsp** [comment]
Arch has updated to Gnome 40, breaking this extension among several others. Is there somewhere the upgrade activity here is being tracked? Happy to contribute,  if I can be pointed to a PR or a branch.

**Comment by PapyElGringo** [comment]
The WIP is tracked theere https://github.com/material-shell/material-shell/pull/632

**Comment by t-rad679** [comment]
Just checking in here. I'd really like to get back in sync with package updates. Is there an ETA on the fix?

**Comment by micahthomas** [comment]
> Just checking in here. I'd really like to get back in sync with package updates. Is there an ETA on the fix?

It's generally not considered polite to ask for ETAs on issues for open source software. The contributors are usually working on projects in their spare time, and have plenty on their plate. Adding more notifications to their plate probably won't help, and I acknowledge me responding to this doesn't either.

If it's important for you to know when it's resolved, use the subscribe button on the issue/pull request.

**Comment by Chrysostomus** [comment]
As a general rule, that is solid. I'd argue that there are sometimes legit reasons for asking about the intented schedule though. It might be necessary for co-ordinating other open source projects that depend on the project in question. For example, if I'm packagaging and distributing material-shell, knowing when I can update the distro to gnome40 stack is very useful. The developer has confirmed that he has started working on it recently, so all is well. Thank you again for the amazing work you do! 🙂

Btw, currently material-shell works just fine on gnome40. There is no major breakage. The biggest issue is that the design clashes with the gnome40 activities view. My current workaround is to run the vertical workspaces extension alongside material-shell. Replacing the activities overview as planned is most likely the ideal solution. If the existing gnome search functionality can be leveraged here, that would be great, though I have no idea if that is feasible.

**Comment by t-rad679** [comment]
Fair enough. Asking for an ETA was intended to be a polite way to add my voice to the people who need the fix, but I'll refrain in the future. Do you have a recommendation on another way to add my +1?

@Chrysostomus What you've described has not been my experience. Material shell is unusably broken for me, and locking dependencies to the previous version of gnome solves the issue. When I get back to my workstation I can provide the specific error message I'm seeing, if that helps.

**Comment by paranoidsp** [comment]
This works great with the latest release!
Closing this.

**Comment by derrix060** [comment]
@Chrysostomus close the issue?  I can also confirm that it is working fine in Gnome 40.  I'm using Fedora 34
